src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -86,6 +89,9 @@ public override async Task<ResourceElement> GetCapabilityStatementOnStartup(Canc
                         }
 
                         _listedCapabilityStatement = _builder.Build().ToResourceElement();
+
+                        _rebuilder = new Thread(RebuildCapabilityStatement);
+                        _rebuilder.Start();
```

Comment:
I like this pattern more:

```csharp
CancellationTokenSource _cancellationTokenSource = new CancellationTokenSource();

//in .ctor
_backgroundLoop = Task.Run(BackgroundLoop, CancellationToken.None);

private async Task BackgroundLoop()
{
      while (!_cancellationTokenSource.IsCancellationRequested)
      {
        // work
      }
}
```

HPaas has an example in ContainerManager

Reply:
Refactored to use this pattern.